### DISCUSSION TITLE
Can I build a new Lambda from an existing servless.yml file

### DISCUSSION THREAD

**Discussion by justageek** [serverless/serverless#10488] [Q&A]
I have an existing serverless.yml file that is nearly identical to needs for a new Lambda function, I'm wondering if I can just copy this yml file, modify the parts of it to define a new function, then deploy and build it, without needing to build anything through the AWS console?

**Comment by MRdgz** [comment]
Yes, if I'm not mistaken. That is assuming that you 
1. Already have an existing AWS account 
2. You are using a CLI that has been configured to use your AWS credentials
3. The credentials you are using have the appropriate permissions to create the needed assets for serverless.

**Comment by pgrzesik** [comment]
Yes @justageek, you can definitely do that. You can also add an additional function definition in an existing configuration file. If you decide to copy configuration, please remember to replace `service` in config - it should be unique.
